rosdep/python.yaml

```diff
@@ -6955,6 +6955,13 @@ python3-kitchen:
   ubuntu:
     '*': [python3-kitchen]
     xenial: null
+python3-kivy:
+  debian:
+    '*': [python3-kivy]
+    buster:
+      pip:
+        packages: [Kivy]
```

Comment:
I'm not sure we want to mix pip and system packages in the same rosdep rule. We can avoid the warning for buster by making the rule:

```
buster: null
```

Reply:
@wjwwood  thoughts?

Reply:
I was following the contribution guide in this case

> Some packages are only available as system packages in newer distributions. In such case, you can specify the default as system package, and pip for older distributions for which the system packages are not available.

Reply:
Ah, cool. I found the section: https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md#pip-mixed-with-system-packages